Jenkinsfile

```diff
@@ -78,7 +78,7 @@ def buildStaticAssets(env, tag) {
 
   sh "npm run build:$env"
   sh 'rm -rf staticAssets && mkdir staticAssets'
-  sh "find build -type f -name '*.png' -delete" // Temp remove all .png assets to speed up upload of static assets (These assets are already avaiable on the CDN)
+  // sh "find build -type f -name '*.png' -delete" // Temp remove all .png assets to speed up upload of static assets (These assets are already avaiable on the CDN)
```

Comment:
Do we need to revert this once the images have been uploaded?

Reply:
Yeah - I'll write an issue for that.  Thanks

Reply:
#7360